webdriver/tests/bidi/web_extension/install/install.py

```diff
@@ -25,7 +25,10 @@ async def test_install_from_path(bidi_session, extension_data):
         }
     )
     try:
-        assert web_extension == extension_data["id"]
+        if extension_data["id"] is None:
+          assert isinstance(web_extension, str)
+        else:
+          assert web_extension == extension_data["id"]
```

Comment:
Mind creating a helper for it so that it can be shared with all the other tests doing the same assertion?

Reply:
it will be only used for the unpacked test. We could add a helper once there are more tests like this.